axlearn/cloud/gcp/system_characteristics.py

```diff
@@ -379,6 +379,28 @@ class _SystemCharacteristics:
         AcceleratorType["TPU"],
         "v5litepod-256",
     ),
+    # v6e
```

Comment:
Is there a reference for these?

Reply:
Yeah here is the link to the PR for the same changes in XPK: https://github.com/google/xpk/pull/179